scripts/index.js

```diff
@@ -1,9 +1,83 @@
-// let parentDiv =document.querySelector(".content");
 
-// let birdCassowary = document.createElement("div");
-// mainContent.classList.add("testdiv");
-// mainContent.textContent = "Hello this is generated via Script";
-// parentDiv.appendChild(mainContent);
+function InfoText(title, images, description) {
+    this.title = title;
+    this.images = images; 
+    this.description = description;
+}
 
-// document.querySelector(".testdiv").style.color = "#fff";
+let planYourDay = new InfoText(
+    "Plan your Day",
+    [
+        "images/index/welcome_page/plan_your_day/zoo_visit_1.jpg",
+        "images/index/welcome_page/plan_your_day/zoo_visit_2.jpg",
+        "images/index/welcome_page/plan_your_day/zoo_visit_3.jpg",
+        "images/index/welcome_page/plan_your_day/zoo_visit_4.jpg"
+    ],
+    "SafariPark is open from 9:30 AM to 5:00 PM, all year except Christmas Day. Child entry is for 3-14 year olds. Free entry for children younger than 3 years old. Children under 14 need to have an adult with them. Please bring your ID, as we offer discounts to students, Community Services card holders, and Gold Card holders. Caregivers of disabled persons are given complimentary entry to the zoo upon presentation of suitable supporting documentation or identification. Please note that SafariPark does not have luggage storage facilities (i.e., lockers). If you have large bags, please chat with one of our Visitor Advisors, who may be able to store them in the front office during your visit.");
 
+let exploreZoo = new InfoText(
+    "Explore the zoo",
+    [
+        "images/index/welcome_page/explore_zoo/playarea_1.jpg",
+        "images/index/welcome_page/explore_zoo/playarea_2.jpg",
+        "images/index/welcome_page/explore_zoo/playarea_3.jpg",
+        "images/index/welcome_page/explore_zoo/playarea_4.jpg"
+    ],
+    "Exploring a zoo is a fascinating journey into the world of wildlife and conservation. From lush green aviaries to aquatic exhibits, every corner unveils unique habitats teeming with life. Visitors can marvel at majestic big cats prowling their enclosures, playful otters gliding through water, or primates showcasing their agility and intelligence. Interactive zones like petting zoos provide hands-on experiences, while informative signage and guided tours offer insights into animal behavior and conservation efforts. Nocturnal exhibits reveal the wonders of creatures active at night, and live feeding sessions let keepers share captivating stories about diets and habits. Relaxing at shaded picnic spots or zoo cafes adds a refreshing touch, making the visit both enjoyable and educational. A day at the zoo is more than entertainment—it's an inspiring opportunity to connect with and protect our planet’s incredible biodiversity. does not have luggage storage facilities i.e. lockers. If you have large bags, please chat."
```

Comment:
"incredible biodiversity. does not have luggage storage facilities i.e. lockers." ???